### ISSUE TITLE
unrecognized arguments: --is_rgbt False

### ISSUE THREAD

**Issue by AnyaTracy** [Dong-Huo/RGB-T-Glass-Segmentation#1]
Hey bro, what should I do if I want to use RGB-only for testing? 
I tried to run the code 'python main.py --rgbt_path xxxx --is_rgbt False --resume xxxx --eval' in readme, but error information 'GETR training and evaluation script: error: unrecognized arguments: --is_rgbt False' appeared.

**Comment by Dong-Huo** [comment]
Thank you for your feedback. I modified the main.py and engine.py, see if it helps.

**Comment by AnyaTracy** [comment]
Thank you for your help. I updated main.py and engine.py and tried to run it, but the problem still exist as 'unrecognized arguments: False'. Maybe we should delete False from 'python main.py --rgbt_path xxxx --is_rgbt False --resume xxxx --eval' ? And maybe the default value of --is_rgbt should be set as False? Because I tried to run main.py with --is_rgbt and without --is_rgbt, both the value of this arguments was True. 
By the way, does RBG-only mean I can test my data without boundary.png, mask.png and temperature.npy?

**Comment by Dong-Huo** [comment]
Yeah, maybe you can just manually change the default value of is_rgbt in main.py as False. 
RGB-only means you do not need the temperature.npy.  boundary.png is never used during the training and testing.  mask.png is for calculating the evaluation metrics.

**Comment by AnyaTracy** [comment]
Thank you for your help again. Maybe it's because I'm stupid, I cannot run RGB-only correctly with only RGB and mask. I set 'is_rgbt' as False and run the program to test my dataset by command below 
'python main.py --rgbt_path RGB-T-glass  --resume checkpoints/checkpoint_rgb.pth --eval ' 
It should be correct because RGB-only only need mask.png and RGB as you said. But it seems like temperature.npy and boundary.png is necessary, here is the error information:
[Errno 2] No such file or directory: 'RGB-T-glass\test\panorama\SUS3333\temperature.npy'
I think about this problem for a long time and still confused about it. I checked the function of evaluate in engine.py, both loop will extract rgb, temperature from data_loader, and the forward function in GETR seems need argument 'temperature'  to run.
